libs/elasticsearch/langchain_elasticsearch/vectorstores.py

```diff
@@ -647,7 +745,7 @@ class ElasticsearchStore(VectorStore):
                 distance_strategy="DOT_PRODUCT"
             )
 
-    """
+    """  # noqa: E501
```

Comment:
```suggestion
    """
```
I think can remove with reformatted docker